### PR TITLE
Hide Rise roles for non Rise users

### PR DESCRIPTION
**Pull request by olegrise**
@ezequielc please review

### CODE REVIEW

File: src/js/svc-userstate.js [hunk 1]
```diff
@@ -510,10 +510,10 @@
           return _state.inRVAFrame;
         },
         isRiseAdmin: function () {
-          return hasRole("sa");
+          return hasRole("sa") && companyState.isRootCompany;
```
Comment (ezequielc) [inline src/js/svc-userstate.js]: @olegrise Is `companyState.isRootCompany` intentionally missing parenthesis? Thanks

**Comment by ezequielc** [comment]
@olegrise Just one question, otherwise looks good. Cheers

**Comment by olegrise** [comment]
@ezequielc good catch. I'm surprised my tests worked. Fixed parenthesis and squashed.

**Comment by ezequielc** [comment]
@olegrise Looks good! :+1: Thanks!

**Comment by olegrise** [comment]
@ezequielc added missing `!` in the `isRootCompany` function. The root company should not have a parent!

**Comment by ezequielc** [comment]
@olegrise Oops, I missed that too. Isn't the field named `parentId`? If that is the case, it might be blank not only null. Thanks

**Comment by olegrise** [comment]
@ezequielc OMG, it's embarrassing. Of course it's `parentId`. Fixed. Thanks.

**Comment by ezequielc** [comment]
@olegrise :) No worries. 
I think you should also check for blank/empty string in `parentId`. I just verified and I got a blank value when using my Rise account. Sorry if I wasn't clear in my previous comment. 
Thanks

**Comment by olegrise** [comment]
@ezequielc if `parentId` is empty string or undefined or null, won't this `!_state.userCompany.parentId` return TRUE?

**Comment by ezequielc** [comment]
@olegrise Sorry, you are right, empty string will work properly. Looks good :+1: . Thanks!

**Comment by olegrise** [comment]
Thanks @ezequielc !

**Comment by olegrise** [comment]
Oops, forgot to squash.

**Comment by olegrise** [comment]
@ezequielc this is tagged as `v1.9.3`

**Comment by ezequielc** [comment]
Thanks @olegrise

**Pull request merged**